libs/ipc/src/dbus_client_iface.cpp

```diff
@@ -14,7 +14,10 @@
 namespace tfc::ipc_ruler {
 
 ipc_manager_client::ipc_manager_client(asio::io_context& ctx)
-    : ipc_manager_client(std::make_shared<sdbusplus::asio::connection>(ctx, tfc::dbus::sd_bus_open_system())) {}
+    : ipc_manager_client(std::make_shared<sdbusplus::asio::connection>(ctx, tfc::dbus::sd_bus_open_system())) {
+  // If the owner of this client does not supply dbus connection we assume we are the sole owner of the connection
+  connection_->request_name(tfc::dbus::make_dbus_process_name().c_str());
```

Comment:
if manager client is spawned by context he makes the connection and requests name, not sure what happens if there are more than one who requests the same name? But I guess it won't happen in current code base

Reply:
> But I guess it won't happen in current code base

Famous last words

Reply:
it throws an error 
```cpp
        if (r < 0)
        {
            throw exception::SdBusError(-r, "sd_bus_request_name");
        }
```
but I don't see a reason catching it because we cannot continue if it fails right?